src/api/game/get/mutations.ts

```diff
@@ -0,0 +1,11 @@
+import { getRequest } from "../../request";
+import { type IGetGamesMutationParams } from "../../types";
+import { type IGame } from "../types";
+import { getGamesByUserIdUrl } from "../urls";
+
+export async function getGamesByUserId({
+  accessToken,
+  id
+}: IGetGamesMutationParams): Promise<{content: IGame[]}> {
+  return await getRequest({ accessToken, url: getGamesByUserIdUrl(id) });
```

Comment:
avoid CRUD-based names on URL, it's understandable from the context that this is a url for this GET action, because you are using it inside GET request.

```suggestion
  return await getRequest({ accessToken, url: gamesByUserId(id) });
```